Fleece/Core/SharedKeys.cc

```diff
@@ -21,13 +21,30 @@
 #include "FleeceException.hh"
 
 
+#ifdef __APPLE__
```

Comment:
This should be in the header, replacing the existing typedef of PlatformString. Here you've created a _second_ typedef of PlatformString, this time outside any namespace, which isn't being used by anything.

Reply:
Note that you will have split the `#include` out into a separate `#ifdef` block at the top of the header, since you don't want to include a header while inside a namespace or class definition...

Reply:
done!